lib/workers/schedule_worker/schedules_builder.rb

```diff
@@ -0,0 +1,41 @@
+class ScheduleWorker < WorkerBase
+  class SchedulesBuilder
+    include Vmdb::Logging
+
+    attr_reader :schedules
+
+    def initialize(scheduler, queue, schedule_category = nil)
+      @scheduler = scheduler
+      @queue = queue
```

Comment:
hm, this probably should not be borrowing the worker's data and doing things with it. i'll try to think of a better way to work this out